lib/core/src/lib/dynamic-chip-list/dynamic-chip-list.module.ts

```diff
@@ -17,21 +17,9 @@
 
 import { NgModule } from '@angular/core';
 import { DynamicChipListComponent } from './dynamic-chip-list.component';
-import { MatChipsModule } from '@angular/material/chips';
-import { MatIconModule } from '@angular/material/icon';
-import { MatButtonModule } from '@angular/material/button';
-import { TranslateModule } from '@ngx-translate/core';
-import { CommonModule } from '@angular/common';
 
 @NgModule({
-    declarations: [DynamicChipListComponent],
-    imports: [
-        MatChipsModule,
-        MatIconModule,
-        MatButtonModule,
-        TranslateModule,
-        CommonModule
-    ],
+    imports: [DynamicChipListComponent],
     exports: [DynamicChipListComponent]
 })
 export class DynamicChipListModule {}
```

Comment:
this whole module is probably not needed anymore, you can import/export this component in the CoreModule instead

Reply:
other applications are dependent on this module, so maybe it will be better to change it there first